content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+lastmod: 2023-11-13
+images: ['localstack-3.0.png']
+contributors: ["LocalStack Team"]
+tags: ['news']
+show_cta_1: true
+leadimage: 'localstack-3.0.png'
+---
+
+{{< img-simple src="localstack-3.0.png" >}}
+
+Today, we are excited to announce **LocalStack 3.0**, our latest major release towards empowering developers with a high-fidelity local cloud development & testing experience. Earlier this year, we unveiled [LocalStack 2.0](https://localstack.cloud/blog/2023-03-29-announcing-localstack-2.0-general-availability/), which brought forth a host of new features and improvements, along with better integration with the AWS tooling ecosystem. With the 3.0 release, we're doubling down on our promise to boosting developer productivity and tackling the issues that often slow down cloud software development and delivery workflows — specifically, simplifying onboarding and eliminating the inefficiencies that lead to prolonged dev&test cycles. Additionally, this release represents a significant milestone in our journey as we move beyond traditional cloud emulation, and introduce innovative features that promise to reshape the way developers construct resilient cloud applications.
+
+Whether you’re a LocalStack user or just getting started on your cloud development journey, we have a lot of exciting features to share with you! With LocalStack, you can now:
+
+- Test disaster recovery simulations such as region failover, DNS failover, and service failures.
+- Explore and progressively tighten security around IAM policies as your application takes shape.
+- Create ephemeral cloud sandboxes for testing your infrastructure locally or on CI pipelines.
+- Activate analytics for LocalStack in CI runs to get insights into your infrastructure test runs.
+- Utilize enhanced, faster AWS services such as S3, Step Functions, DynamoDB and ElastiCache.
+- Get started with a fully-local developer environment with the new LocalStack Desktop Application.
+
+And much more! LocalStack makes it easy to build & test local-first cloud applications, all while improving the **Inner Dev Loop**. We are excited to share our recent developments in this release, and discuss how you can get started with them!
+
+## What’s new in LocalStack 3.0?
+
+In previous releases, we supplemented our cloud emulation capabilities with parity, performance, and interoperability. In LocalStack 3.0, our core areas of focus has been — **resilience**, **efficiency**, and **flexibility**. With LocalStack, we aim to power your local development productivity and cover the most sophisticated enterprise use cases. Collectively, the new features in the release won't just cut away your slow *code → deploy → test → redeploy → ...* loops, but also help your team to confidently ship faster and more secure cloud applications. You can explore *all* the new features by upgrading your LocalStack setup or downloading LocalStack 3.0 today! [Get your free LocalStack account](https://app.localstack.cloud/sign-up).
+
```

Comment:
with the heading, and because we already have the intro paragraph on top of the post, I'd probably skip this paragraph when reading and head straight to the list of 'whats new'. does not hurt to have it in there, but something to consider for future posts.